### PR TITLE
Fix issues generating v4 packages

### PR DESCRIPTION
**Pull request by larioj**
Tried to generate some v4 packages, and found these issues.

### CODE REVIEW

File: scripts/gen-universe.py [hunk 1]
```diff
@@ -588,6 +591,8 @@ def v4_to_v3_package(v4_package):
     package = copy.deepcopy(v4_package)
     package.pop('upgradesFrom', None)
     package.pop('downgradesTo', None)
+    package['command'] = None
```
Comment (cruhland) [inline scripts/gen-universe.py]: Is `command` required for v3 packages? Why not just leave it unspecified?

File: scripts/gen-universe.py [hunk 2]
```diff
@@ -368,6 +368,9 @@ def generate_package(
         package['config'] = config
     package['command'] = command
 
+    if package['packagingVersion'] == "4.0":
+        package.pop('command', None)
+
```
Comment (jsancio) [inline scripts/gen-universe.py]:
```python
if command:
  package['command'] = command
```

File: scripts/gen-universe.py (hunk 1 above)
Comment (jsancio) [inline scripts/gen-universe.py]: I think we can remove this line.

File: scripts/gen-universe.py [hunk 3]
```diff
@@ -642,7 +647,7 @@ def _validate_repo(file_path, version):
     else:
         repo_version = 'v3'
 
-    validator = jsonschema.Draft4Validator(_load_jsonschema('v3'))
+    validator = jsonschema.Draft4Validator(_load_jsonschema(repo_version))
```
Comment (jsancio) [inline scripts/gen-universe.py]: :+1:

File: scripts/gen-universe.py (hunk 2 above)
Comment (larioj) [inline scripts/gen-universe.py]: 👍

**Comment by jsancio** [comment]
@larioj I also have this change:
```
diff --git a/repo/meta/schema/v4-repo-schema.json b/repo/meta/schema/v4-repo-schema.json
index 35593c3..2a26772 100644
--- a/repo/meta/schema/v4-repo-schema.json
+++ b/repo/meta/schema/v4-repo-schema.json
@@ -591,9 +591,6 @@
         },
         "config": {
           "$ref": "#/definitions/config"
-        },
-        "command": {
-          "$ref": "#/definitions/command"
         }
       },
       "required": [
```

**Pull request merged**